.vscode/launch.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
I don't think this file should be included since it set up by the idf extension?